interfaces/builtin/mount_observe.go

```diff
@@ -49,11 +49,8 @@ const mountObserveConnectedPlugSecComp = `
 # restricted because it gives privileged read access to mount arguments and
 # should only be used with trusted apps.
 
-quotactl Q_GETQUOTA - - -
-quotactl Q_GETINFO - - -
-quotactl Q_GETFMT - - -
-quotactl Q_XGETQUOTA - - -
-quotactl Q_XGETQSTAT - - -
+# FIXME: restore quotactl with parameter filtering once # snap-confine can read
+this syntax.  # See LP:#1662489 for context.
```

Comment:
You have invalid syntax here ('this' wrapped around without a comment). Also, rather then removing the rules, can you just comment them out? Eg:
```
# FIXME: restore quotactl with parameter filtering once snap-confine can read
# this syntax. See LP:#1662489 for context.
#quotactl Q_GETQUOTA - - -
#quotactl Q_GETINFO - - -
#quotactl Q_GETFMT - - -
#quotactl Q_XGETQUOTA - - -
#quotactl Q_XGETQSTAT - - -

```